docs/throttling.md

```diff
@@ -67,9 +70,61 @@ For more information and a complete list of features visit the documentation on
 # Enable system traffic throttling
 throttle 3gfast
 
-# Run Lighthouse with its own throttling disabled
-lighthouse --throttling-method=provided # ...
+# Run Lighthouse with its own network throttling disabled (while leaving CPU throttling)
+lighthouse --throttling-method=devtools \
+  --throttling.requestLatencyMs=0 \
+  --throttling.downloadThroughputKbps=0 \
+  --throttling.uploadThroughputKbps=0 \
+  https://example.com
 
 # Disable the traffic throttling once you see "Gathering trace"
 throttle --stop
 ```
+
+# CPU Throttling
+
+Lighthouse applies CPU throttling to emulate a mid-tier mobile device even when run on far more powerful desktop hardware.
+
+## Types of CPU Throttling
+
+Within web performance testing, there are two typical styles of CPU throttling:
+
+1. **_Simulated throttling_**, which Lighthouse uses by **default**, uses a simulation of a page load, based on the data observed in the initial unthrottled load. This approach makes it very fast. However, due to the imperfect nature of predicting alternate execution paths, there is inherent inaccuracy that is summarized in this doc: [Lighthouse Metric Variability and Accuracy](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit). The TLDR: while it's fairly accurate for most circumstances, it suffers from edge cases and a deep investigation to performance should use _applied_ CPU throttling tools.
+1. **_Applied throttling_** , also called _`devtools` throttling_ in Lighthouse configuration. This method actually interrupts execution of CPU work at periodic intervals to emulate a slower processor. It is [fairly accurate](https://docs.google.com/document/d/1jGHeGjjjzfTAE2WHXipKF3aqwF2bFA6r0B877nFtBpc/edit) and much easier than obtaining target hardware. The same underlying principle can be used by [linux cgroups](https://www.kernel.org/doc/html/latest/scheduler/sched-bwc.html) to throttle any process, not just the browser. Other tools like [WebPageTest use applied CPU throttling](https://github.com/WPO-Foundation/wptagent/commit/f7fe0d6b5b01bd1b042a1fe3144c68a6bff846a6) offered by DevTools.
+
+## Calibrating Multipliers
+
+Unlike network throttling where objective criteria like RTT and throughput allow targeting of a specific environment, CPU throttling is expressed relative to the performance of the host device. This poses challenges to [variability in results across devices](./variability.md), so it's important to calibrate your device before attempting to compare different reports.
+
+Lighthouse computes and saves a `benchmarkIndex` as a rough approximation of the host device's CPU performance with every report. You can find this value under the title "CPU/Memory Power" at the bottom of every Lighthouse report.
```

Comment:
oh.. last thing . do you want to add a note in this doc that we tweaked benchmarkIndex calculation in lh 6.3?

Reply:
```suggestion
Lighthouse computes and saves a `benchmarkIndex` as a rough approximation of the host device's CPU performance with every report. You can find this value under the title "CPU/Memory Power" at the bottom of the Lighthouse report.

**NOTE:** In Lighthouse 6.3 BenchmarkIndex changed its definition to better align with changes in Chrome 86. Benchmark index values prior to 6.3 and Chrome 86 may differ.
```